examples/jvm/jvm-return-example.yaml

```diff
@@ -16,5 +16,3 @@ spec:
     labelSelectors:
       app: springboot-jvmchaos-demo
   duration: "50s"
-  scheduler:
-    cron: "@every 1m"
```

Comment:
Why delete `cron` in the example?

Reply:
Sry I forgot to upload the yaml

Reply:
`cron` part is delegated to `schedule` in nirvana. I feel it unnecessary to test it here.

Reply:
got it.maybe we should update the other examples.

Reply:
An issue #1887 has been added.